packages/babel-preset-gatsby-package/lib/__tests__/fixtures/keep-dynamic-import/with-override/output.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+exports.__esModule = true;
+exports.someFunction2 = someFunction2;
+
+async function someFunction2() {
+  return await import(`path`);
```

Comment:
this file was marked to keep dynamic import in https://github.com/gatsbyjs/gatsby/blob/906e491732c14e715e3258f4a676e23d162b62f8/packages/babel-preset-gatsby-package/lib/__tests__/fixtures/keep-dynamic-import/options.js#L9-L14 and it in fact keep that instead of doing transformation